tests/test_norwegianblue.py

```diff
@@ -406,6 +406,23 @@ def test_404(self) -> None:
         ):
             norwegianblue.norwegianblue(product="androd")
 
+    @respx.mock
```

Comment:
Rather than adding a whole new case, let's parametrise the previous one and add it there:

```python
    @respx.mock
    @pytest.mark.parametrize(
        "product, expected",
        [
            (
                "androd",
                r"Product 'androd' not found, run 'eol all' for list\. "
                r"Did you mean: 'android'?",
            ),
            ("julia", r"Product 'julia' not found, run 'eol all' for list\."),
        ],
    )
    def test_404(self, product, expected) -> None:
        # Arrange
        mocked_url = f"https://endoflife.date/api/{product}.json"
        respx.get(mocked_url).respond(status_code=404)

        mocked_url = "https://endoflife.date/api/all.json"
        mocked_response = SAMPLE_RESPONSE_ALL_JSON
        respx.get(mocked_url).respond(content=mocked_response)

        # Act / Assert
        with pytest.raises(ValueError, match=expected):
            norwegianblue.norwegianblue(product=product)
```

Reply:
Thanks!